R/AutoGeneratedDefinitions.R

```diff
@@ -1565,7 +1566,26 @@ detectPathwaysByName <- function(pattern, negate = FALSE, baseUrl) {
 #'                       baseUrl = "http://server.org:80/WebAPI")
 #' }
 #' @export
-postPathwayDefinition <- function(name, pathwayDefinition, baseUrl) {
+postPathwayDefinition <- function(name, pathwayDefinition, baseUrl, postCohorts = TRUE) {
```

Comment:
The AutoGeneratedDefinitions.R should be generated automatically in packageMaintenance 

see https://github.com/OHDSI/ROhdsiWebApi/blob/3e09d7350984c4ec9b4a93678b3492b5fcbf67ef/R/AutoGeneratedDefinitions.R#L18

and from [here](https://github.com/OHDSI/ROhdsiWebApi/blob/3e09d7350984c4ec9b4a93678b3492b5fcbf67ef/extras/PackageMaintenance.R#L52) 

I cant accept this change as written - because it did use existing framework and would break at next package update.

Reply:
Ah, I did wonder what the filename referred to.

So I guess the functionality here can be added to the main postDefinitions function early on (i.e. before the main `httr::POST`?

Reply:
I have moved this functionality from ```R/AutoGeneratedDefinitions.R``` (which is now generated directly using ```PackageMaintanence.R``` into ```postDefinition```:

https://github.com/hms1/ROhdsiWebApi/blob/8a6a7ab6f61fad6c3b978cf4af7ba67d7ee70935/R/PostDefinition.R#L77-L89